Assets/Scripts/CoinManager.cs

```diff
@@ -67,8 +67,11 @@ private void AddCoin(int viewId)
 
         private IEnumerator RespawnTimer()
         {
-            yield return new WaitForSeconds(SecondsToRespawn);
-            mPhotonView.RPC("Respawn", PhotonTargets.All);
+            if (SecondsToRespawn >= 0)

```

Comment:
we can set seconds to respawn below 0 if we don't want the coin to respawn (like in the ready room)